docs/hub/model-cards.md

```diff
@@ -15,10 +15,16 @@ The model card should describe:
 - its intended uses & potential limitations, including biases and ethical considerations as detailed in [Mitchell, 2018](https://arxiv.org/abs/1810.03993)
 - the training params and experimental info (you can embed or link to an experiment tracking platform for reference)
 - which datasets were used to train your model
-- your evaluation results
+- the model's evaluation results
 
 The model card template is available [here](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md).
 
+How to fill out each section of the model card is described in [the Annotated Model Card](https://huggingface.co/docs/hub/model-card-annotated).
+
+Model Cards on the Hub have two key parts, with overlapping information:
```

Comment:
Thanks! I would suggest bringing some of the content in line 30 to this section (and then remove line 30)

> A model repo will render its `README.md` as a model card. The model card is a [Markdown](https://en.wikipedia.org/wiki/Markdown) file, with a [YAML](https://en.wikipedia.org/wiki/YAML) section at the top that contains metadata about the model.

Reply:
Hmm, so the goal here is to have the critical links without bringing in a lot more wording/writing. 
As-is, I think I disagree with this suggestion, as it would bury the links amid more wording.

Reply:
> One possibility is to move ## Model card metadata down to follow the paragraph on line 30 (so, move it to line 31); and possibly to add to line 30 "...and [more open/long-form/descriptive] text throughout the README.md"

I'm not sure I see the difference between this and what I suggested above. I was suggesting to have

```
- [Metadata](#model-card-metadata)
- [Text descriptions](#model-card-text)

A model repo will render its `README.md` as a model card. The model card is a [Markdown](https://en.wikipedia.org/wiki/Markdown) file, with a [YAML](https://en.wikipedia.org/wiki/YAML) section at the top that contains metadata about the model.

## Model card metadata
```

Reply:
K, I'm going to stop working on this PR then.
"You can find a model card as the `README.md` file in any model repo." and "A model repo will render its `README.md` as a model card." is a bit redundant, so perhaps someone else will address at some point.
Otherwise, please feel free to approve to merge as-is.

Reply:
Let's go ahead with this PR